compiler/build.gradle

```diff
@@ -100,7 +100,9 @@ model {
                 } else if (osdetector.os == "windows") {
                     linker.args "-static", "-lprotoc", "-lprotobuf", "-static-libgcc", "-static-libstdc++",
                             "-s"
-                } else {
+		} else if (osdetector.arch == "ppcle_64") {
+			linker.args "-Wl,-Bstatic", "-lprotoc", "-lprotobuf", "-Wl,-Bdynamic", "-lpthread", "-s"
```

Comment:
Pls align the `linker.args...` statement with the one on line 101 to start at the same column

Reply:
@sanjaypujare have fixed these too, thanks .